README.md

```diff
@@ -130,23 +130,33 @@ $harvestService = $greenhouseService->getHarvestService();
 ?>
 ```
 
-Via the Harvest service, you can interact with any Harvest methods outlined in the Greenhouse Harvest docs.  Harvest URLs fit mostly in to one of the following three formats:
+Via the Harvest service, you can interact with any Harvest methods outlined in the Greenhouse Harvest docs.  Harvest URLs fit mostly in to one of the following five formats:
 
 1. `https://harvest.greenhouse.io/v1/<object>`: This is the most common URL format for GET methods in Greenhouse.  For endpoints in this format, the method will look like `$harvestService->getObject()`.  Examples of this are `$harvestService->getJobs()` or `$harvestService->getCandidates()`
 2. `https://harvest.greenhouse.io/v1/<object>/<object_id>`: This will get the object with the given ID.  This is expected to only return or operate on one object.  The ID will always be supplied by an parameter array with a key named `id`.  For instance: `$harvestService->getCandidate($parameters);`
 3. `https://harvest.greenhouse.io/v1/<object>/<object_id>/<sub_object>`: URLs in this format usually mean that you want to get all the sub_objects for the object with the object id.  Examples of this are `$harvestService->getJobStagesForJob(array('id' => 123))` and `$harvestService->getOffersForApplication(array('id' => 123))`
-4. Some method calls and URLs do not exactly fit this format, but the methods were named as close to fitting that format as possible.  These include:
+4. `https://harvest.greenhouse.io/v1/<object>/<object_id>/<sub_object>/<sub_object_id>`: URLs in this format usually mean you are performing an operation on an individual sub-object.  An example of this is `$harvestService->deleteTagsForCandidate(array('id' => 123, 'second_id' => 234))`
+5. `https://harvest.greenhouse.io/v1/<object>/<opbject_id>/<sub_object>/<qualifier>`: Urls in this format usually mean you are trying to act on a limited universe of a type of sub-object.  An example of this is `$harvestService->deletePermissionForJobForUser(array('id' => 123));` which deletes the designated permission on a job from a user.
+6. Some method calls and URLs do not exactly fit this format, but the methods were named as close to fitting that format as possible.  These include:
```

Comment:
Sorry to nitpick, but, why is this in the list? Shouldn't it either say "six formats," or have this in a separate paragraph?